syntax = "proto3";

package protobufs;

message ParamsRequest {
    bytes paraRequest = 1;
}

// normalized training data
message ParamsResponse {
    bytes paraResponse = 2;
}

// service method
service MonaiFLService{
  rpc ParamTransfer(ParamsRequest) returns (ParamsResponse);
}